src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/TimeoutTests.cs

```diff
@@ -296,5 +308,18 @@ public static void AiaFetchTimeout()
             chain.ChainElements.OfType<X509ChainElement>().
                 Single(e => e.Certificate.Thumbprint == thumbprint).
                 ChainElementStatus.Aggregate((X509ChainStatusFlags)0, (a, e) => a | e.Status);
+
+        private static bool Retry(Func<bool> body, int times = 3)
```

Comment:
@vcsjones FYI there's already a `RetryHelper` in TestUtilities that implements similar retry, it could be extended to cover the `bool` case: https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/TestUtilities/System/RetryHelper.cs